src/tests/ftest/pool/multi_server_create_delete_test.py

```diff
@@ -25,41 +25,35 @@
 
 import os
 import traceback
-import json
-
 from avocado.utils import process
-from apricot import Test
-
+from apricot import TestWithoutServer, skipForTicket
```

Comment:
(pylint-no-name-in-module) No name 'TestWithoutServer' in module 'apricot'
(pylint-unused-import) Unused TestWithoutServer imported from apricot